src/Analyzers/CSharp/CodeFixes/UseImplicitOrExplicitType/UseExplicitTypeCodeFixProvider.cs

```diff
@@ -164,6 +166,83 @@ private static ExpressionSyntax GenerateTupleDeclaration(ITypeSymbol typeSymbol,
                 .WithTrailingTrivia(parensDesignation.GetTrailingTrivia());
         }
 
+        private static async Task HandleForEachStatementAsync(Document document, SyntaxEditor editor, ForEachStatementSyntax forEach, CancellationToken cancellationToken)
```

Comment:
>HandleForEachStatementAsync [](http://example.com/codeflow?start=34&length=27)

Is this method used?

Reply:
They should be, looks like I missed part of the port when merging things. CI should fail the new tests without using this